ripd/Makefile

```diff
@@ -0,0 +1,10 @@
+all: ALWAYS
+	@$(MAKE) -s -C .. ripd/ripd
```

Comment:
What is the secret behind backing off the current directory and getting back in?